web-ui/src/api/birthdayanniversary.js

```diff
@@ -4,23 +4,33 @@ const anniversaryReportUrl = "/services/reports/anniversaries";
 const birthdayReportUrl = "/services/reports/birthdays";
 const celebrationsToday = "/services/today";
 
-export const getAnniversary = async (month, cookie) => {
```

Comment:
I had to create the branch for this PR from the feature-2209-birthdays-bug branch because I needed those fixes for the changes to work. So some of the changes here are really from that branch.